packages/server/lib/controllers/files.js

```diff
@@ -85,17 +84,15 @@ module.exports = {
       if (spec === '__all') {
         debug('returning all specs')
 
-        return specsUtil.find(config)
+        return specsUtil.default.findSpecs(config)
         .then(R.tap((specs) => {
```

Comment:
Would love to rip out Ramda whenever we have the opportunity to as well

Reply:
Lachlan and I were just talking about that. 🪓